src/sentry/api/endpoints/organization_projects_experiment.py

```diff
@@ -188,5 +188,7 @@ def post(self, request: Request, organization: Organization) -> Response:
             "created team through project creation flow",
             extra={"team_slug": default_team_slug, "project_slug": project_name},
         )
+        serialized_response = serialize(project, request.user)
+        serialized_response["team_slug"] = team.slug
 
-        return Response(serialize(project, request.user), status=201)
+        return Response(serialized_response, status=201)
```

Comment:
what is the difference between serializing response and passing as text field. For example, why do we serialize project in response but not team?

Reply:
We serialize the project in the response b/c the primary purpose of this API is to create a project. We need to add the `team_slug` in the resp as well b/c it's required in the toast we're displaying to the user. The toast only requires the `team_slug` so we don't need to serialize the rest of team